### DISCUSSION TITLE
在挂着clash代理时软件加载不出来

### DISCUSSION THREAD

**Discussion by KingPenguinMan** [ywmoyue/biliuwp-lite#616] [Q&A] [answered]
### Bug 描述

如题所示，一开始我以为是没有加api的原因，尝试后问题并没有解决

### 操作步骤

```plain text
打开代理，进入软件
```

### 期望结果

在挂起代理时进入软件也可以正常使用

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by KingPenguinMan** [comment]
问题已解决，我的问题

**Comment by Yumeyo23** [comment]
这好像是UWP程序通病，开着Clash应该不止他不能用，其他UWP比如Store之类的应该都连不上网了吧，我也是因为这个原因，直接换软路由了

**Reply by Yumeyo23** [reply]
所以clash里面会有一个UWP程序解锁（？）大概这个名字吧，可以试试那个有无用，总之在我的印象里是所有UWP都没法开着代理联网，现在换软路由了哈哈，所有设备都免开系统代理了

**Reply by KingPenguinMan** [reply, marked as answer]
其实我已经解决了，在软件里就有一个教程，是直接用fiddler直接勾选uwp进行隔离的，然后就可以正常使用了，把Store隔离了也可以正常使用哈哈
